src/rules/no-loud-comments/__tests__/index.js

```diff
@@ -0,0 +1,45 @@
+import rule, { ruleName, messages } from "..";
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  syntax: "scss",
+
+  accept: [
+    {
+      code: "// comment",
+      description: "Double slash comments"
+    }
+  ],
+  reject: [
+    {
+      code: `
+      /* comment */
+    `,
+      description: "One line with optional ending",
+      message: messages.expected
```

Comment:
The tests are using `messages.expected`, but the rule is exposing `messages.rejected`.